src/config.py

```diff
@@ -6,5 +6,26 @@
 
 TOKEN = os.environ.get("TOKEN")  # required
 DEBUG = os.environ.get("DEBUG", False)
+UID_MAP = os.environ.get("UID_MAP")
```

Comment:
I also think the env variable should be changed to `DISCORD_UID_MAP` to avoid confusion